contrib/bash/polybar

```diff
@@ -30,11 +30,11 @@ _polybar() {
 	local options='-h --help
 	               -v --version
 	               -l --log=
-	               -q --quiet
-	               -c --config=
+	               -q --quiet -c --config=
```

Comment:
Is this still the same thing as before (I'm not a bash expert)?

Reply:
That wasn't intentional, will upload a fixed version